src/tools/PlaywrightSharp.Tooling/ApiChecker.cs

```diff
@@ -26,7 +26,7 @@ public bool Execute()
             var assembly = Assembly.LoadFrom(AssemblyPath);
 
             var report = new StringBuilder("<html><body><ul>");
-            string json = File.ReadAllText(Path.Combine(BasePath, "src", "PlaywrightSharp", "runtimes", "api.json"));
+            string json = File.ReadAllText(Path.Combine(BasePath, "src", "PlaywrightSharp", ".playwright", "api.json"));
```

Comment:
This will be the path inside the project. Shouldn't it be `Drivers`?
The same applies for the other changes in this file.